src/components/header/Hero.js

```diff
@@ -27,32 +33,32 @@ const Hero = ({ bounceInAnimation, executeScroll }) => {
                     Next generation DEX crypto trading:
                     <br />
                     <br />
-                    NO Swap Fees...
+                    UI for interacting with KROMatika smart contracts
                     <br />
```

Comment:
@sNnickK I meant all of them. If we delete No Swap Fee... and replace with something that does not say swap fee then we lose important piece of information. So don't remote information, feel free to add but don't remove them